diagnostics/collect-wsl-logs.ps1

```diff
@@ -131,86 +131,56 @@ if ($Dump)
 }
 
 # Collect networking state relevant for WSL
-$wslNetworkLog = "$folder/wsl_network_snapshot.log"
 $stdOutLog = "$folder/wsl_network_stdout.log"
 $stdErrLog = "$folder/wsl_network_stderr.log"
 
+# Using a try/catch as some of the commands below do not exist on some OS versions
 try
 {
-    Write-Output "WSL Net Snapshot" | Out-File -FilePath $wslNetworkLog
+    Write-Output "Get-NetAdapter" | Out-File -FilePath "$folder/Get-NetAdapter.log" -Append
```

Comment:
Given that the file name describes its content, I think we can remove the strings as well to simplify things

Reply:
agreed will update for this and the other commands